<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTransition</class>
 <widget class="QDialog" name="DialogTransition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nueva Transición</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Símbolo de entrada</string>
     </property>
     <property name="buddy">
      <cstring>cb_readSymbol</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Nuevo Estado</string>
     </property>
     <property name="buddy">
      <cstring>cb_targetState</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Símbolo de salida</string>
     </property>
     <property name="buddy">
      <cstring>cb_writeSymbol</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Movimiento</string>
     </property>
     <property name="buddy">
      <cstring>cb_movement</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Estado</string>
     </property>
     <property name="buddy">
      <cstring>cb_State</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="cb_readSymbol"/>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="cb_targetState"/>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="cb_writeSymbol"/>
   </item>
   <item row="3" column="5">
    <widget class="QComboBox" name="cb_movement">
     <item>
      <property name="text">
       <string>I</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>N</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Ingresar nueva transición</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cb_State">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_State</tabstop>
  <tabstop>cb_readSymbol</tabstop>
  <tabstop>cb_targetState</tabstop>
  <tabstop>cb_writeSymbol</tabstop>
  <tabstop>cb_movement</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogTransition</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogTransition</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
